### PR TITLE
Galactic search selector

### PR DESCRIPTION
**Pull request by LQ1234**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
   private AutoCommandInterface m_autonomousCommand;
   private RobotContainer m_robotContainer;
-  SendableChooser<AutoCommandInterface> chosenAuto = new SendableChooser<>();
+  public SendableChooser<AutoCommandInterface> chosenAuto = new SendableChooser<>();
```
Comment (prensing) [inline src/main/java/frc/robot/Robot.java:32]: this should not be public

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -177,7 +183,14 @@ public void autonomousInit() {
    * This function is called periodically during autonomous.
    */
   @Override
-  public void autonomousPeriodic() {
+  public void autonomousPeriodic() {// if the selected auto changed, schedule it
+    AutoCommandInterface autoCommandInterface = chosenAuto.getSelected();
+    if (autoCommandInterface != null && autoCommandInterface != m_autonomousCommand) {
+      m_autonomousCommand.cancel();
+      m_robotContainer.robotDrive.setPose(autoCommandInterface.getInitialPose());
```
Comment (prensing) [inline src/main/java/frc/robot/Robot.java:190]: I don't think any of this belongs in autoPeriodic().

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -131,6 +134,9 @@ public void disabledInit() {
       m_robotContainer.robotDrive.setIdleMode(IdleMode.kCoast);
       m_robotContainer.climber.coastWinch();
     }
+    if (m_autonomousCommand != null) {
+      m_robotContainer.robotDrive.tankDriveVolts(0, 0);
+    }
```
Comment (prensing) [inline src/main/java/frc/robot/Robot.java:139]: This block ought to be near the top of disabledInit, so it happens immediately. You need to set m_autonomousCommand to null so it does not keep doing this.

**Review by prensing — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (coachFitz0807) [reply src/main/java/frc/robot/Robot.java:32]: Concur

File: src/main/java/frc/robot/Robot.java (hunk 3 above)
Comment (coachFitz0807) [reply src/main/java/frc/robot/Robot.java:139]: Concur

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (coachFitz0807) [reply src/main/java/frc/robot/Robot.java:190]: Concur. There is no way to change the command in Autonomous mode.

**Review by coachFitz0807 — CHANGES_REQUESTED** [review]
I concur with Paul's suggestions.

**Review by prensing — APPROVED** [review]

**Review by coachFitz0807 — APPROVED** [review]

**Pull request merged**